modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -4,10 +4,43 @@
 <template>
   <div class="apos-schema">
     <div
-      v-for="field in schema" :key="field.name"
+      v-for="(field, index) in schemaToParse"
+      :key="`${field.name}${index}`"
       :data-apos-field="field.name"
     >
+      <div
+        v-if="field.type === 'alignedFields'"
```

Comment:
This type could be renamed, the system for making fields inline could be rethink, but it wasn't the purpose of this PR and it's where I lost a lot of time. As well as for grouped fields in relationships, that does not exist for now.